pySDC/implementations/problem_classes/HeatEquation_ND_FD_forced_periodic.py

```diff
@@ -1,6 +1,8 @@
+import time
+
 import numpy as np
 import scipy.sparse as sp
-from scipy.sparse.linalg import gmres, spsolve
+from scipy.sparse.linalg import spsolve, cg  # , gmres
 
```

Comment:
The only thing you should change here is using cg. Please remove the other changes.